e2e/suite.go

```diff
@@ -92,11 +94,15 @@ func Run(t *testing.T) {
 	defer os.Remove(imagePath)
 
 	// Start registry for tests
-	singularitye2e.PrepRegistry(t)
-	defer singularitye2e.KillRegistry(t)
+	//	singularitye2e.PrepRegistry(t)
+	//	defer singularitye2e.KillRegistry(t)
 
 	// RunE2ETests by functionality
 
+	t.Run("INSPECT", singularityinspect.RunE2ETests)
+
+	return
```

Comment:
accident, I guess?

Reply:
Yeah, I just did that to "disable" the other tests, so its quicker :+1: